### ISSUE TITLE
[MM Version 1.8.9] Soundcloud Songs not Searching

### ISSUE THREAD

**Issue by dxrthvxder** [MC-U-Team/Music-Player#54]
Most Songs on SoundCloud work, but some songs not work, and when you look them up, it says "something went wrong when looking up the track"

**To Reproduce**
1. Go to playlist song lookup
2. Do this search some songs on SoundCloud such as "iSpy by Kyle and Lil Yachty"

When I looked this song up, I expected it to show all the results, like the other songs I looked up.

![Capturesong](https://user-images.githubusercontent.com/66187603/83657977-995aea00-a58f-11ea-816a-1b8862b62842.PNG)

**Versions**
- Mod version: 2.0.3.70
- Minecraft version: 1.8.9
- Forge version: unkown
- UTeamCore version: unkown

**Comment by HyCraftHD** [comment]
Please send me the log.
Furthermore what have you done that the language file does not work?

**Comment by dxrthvxder** [comment]
I don't know what the language file is, sorry.

**Comment by dxrthvxder** [comment]
And it never crashed. The red words just showed up, and nothing else happened.

**Comment by HyCraftHD** [comment]
Send me the log. Its under logs/debug.log or logs/latest.log

**Comment by dxrthvxder** [comment]
[debug.log](https://github.com/MC-U-Team/Music-Player/files/4724753/debug.log)

**Comment by dxrthvxder** [comment]
Here, this is it right?

**Comment by HyCraftHD** [comment]
No. This log says you don't have music player installed.
Please start minecraft with the musicplayer mod and uteamcore and then try to get the bug you described above. If the bug happens send me the logs/debug.log file please

**Comment by dxrthvxder** [comment]
Yes, I did start it with the mod

**Comment by dxrthvxder** [comment]
The file I sent was loaded with the mod

**Comment by HyCraftHD** [comment]
This here are the only mods installed: 
![grafik](https://user-images.githubusercontent.com/7681220/83665831-46535800-a5cc-11ea-9043-b4c64d2fe3da.png)

**Comment by HyCraftHD** [comment]
So no that must be the wrong file

**Comment by dxrthvxder** [comment]
I will restart Minecraft and send it again.

**Comment by dxrthvxder** [comment]
Oh ok

**Comment by dxrthvxder** [comment]
Yeah I'm gonna send you the file again when I restart Minecraft

**Comment by dxrthvxder** [comment]
Yep, here it is

**Comment by dxrthvxder** [comment]
Right now I have the music player mod and refraction's summer mod folder
installed

**Comment by dxrthvxder** [comment]
I removed a few mods out of the folder because they were incompatible, and
I will send you my mod folder.

**Comment by HyCraftHD** [comment]
I don't see any log. Maybe its because of the email reply you used?

**Comment by dxrthvxder** [comment]
Oh, I will resend it.

**Comment by dxrthvxder** [comment]
Did that work?

**Comment by HyCraftHD** [comment]
No I think email attachments do not work. You can upload the log to pastebin or just use the github page to upload it

**Comment by dxrthvxder** [comment]
Ok.

**Comment by dxrthvxder** [comment]
[1591203719530_debug.log](https://github.com/MC-U-Team/Music-Player/files/4724983/1591203719530_debug.log)

**Comment by dxrthvxder** [comment]
Here it is.

**Comment by HyCraftHD** [comment]
This is the same log as the last one.
You loaded only these mods: 
![grafik](https://user-images.githubusercontent.com/7681220/83667441-aea33900-a5ce-11ea-81a0-62f40569ec84.png)
Furthermore you said before that its minecraft version 1.8.9, but the log says you are starting 1.12.2

**Comment by dxrthvxder** [comment]
And here is my mods folder,

  (1.8.9) IngameServerSwitcher.jar
<https://drive.google.com/file/d/1YX9MRhm8bSLy1tnAPXom_CKj7PjDTBZR/view?usp=drivesdk>

  [1.8.9] Perspective Mod-3.0.jar
<https://drive.google.com/file/d/10F_aIZiot8SJir0HfqPwd2rp2ZzYIexE/view?usp=drivesdk>

  [1.8.9] Powns Coords HUD - 1.0.jar
<https://drive.google.com/file/d/1E_aCwcZAWWYNPrROBtJOQtyVlDnmjRd5/view?usp=drivesdk>

  AutoGG-2.0.4 (1.8.9).jar
<https://drive.google.com/file/d/1ezlByFVadoDiSl8DrfX-qbP1R85qbhNO/view?usp=drivesdk>

  AutoWho-1.0.jar
<https://drive.google.com/file/d/1Cctme18O1f9RdOzJUlghjixiLZj4iZGR/view?usp=drivesdk>

  LabyMod-3.jar
<https://drive.google.com/file/d/1RkowA9I6GHbypZk92ec-d3-FG5G7FMNQ/view?usp=drivesdk>

  MemoryFix-0.3.jar
<https://drive.google.com/file/d/1lzukR-GexE3cGCTrluNH0mguX9_gXhDM/view?usp=drivesdk>

  MouseDelayFix-1.0.jar
<https://drive.google.com/file/d/1ENK1PI2i1z8nTTy8RZi98OOymCQ-4l33/view?usp=drivesdk>

  music_player-1.8.9-2.0.3.70.jar
<https://drive.google.com/file/d/1bnjmje9Y3WIyfDsOU4a6NnhhRoJTuMCp/view?usp=drivesdk>

  OldAnimationsMod v2.4.2 FORGE MC1.8.9.jar
<https://drive.google.com/file/d/1EkshzMD_X2hEWuMCkOiCu8jfDjE-UCRx/view?usp=drivesdk>

  OptiFine_1.8.9_HD_U_I3.jar
<https://drive.google.com/file/d/1fPXTR2z_vBOJdvOGSmeFvbeBneREUd2b/view?usp=drivesdk>

  PlayerAPI-1.8.9-1.0.jar
<https://drive.google.com/file/d/1XjuTxjnWRSGBWSsEYMM-ZkZ0-F3tYadR/view?usp=drivesdk>

  Quickplay Premium 1.8.9-1.0.1.jar
<https://drive.google.com/file/d/1xj5kcDjTta2lQZO_M8NXE2p1hbnkTnBs/view?usp=drivesdk>

  Quickplay-1.8.9-2.0.3.jar
<https://drive.google.com/file/d/10PcT9zQICHgB5kqFVfnDzJ7FdYRfIFl4/view?usp=drivesdk>

  The 5zig Mod v3.11.7 for Minecraft 1.8.9.jar
<https://drive.google.com/file/d/1RvCYtFtDk21oIHLWcEmerMTM7Hwpr2HQ/view?usp=drivesdk>

  TimeChanger-1.0 (1.8.9).jar
<https://drive.google.com/file/d/1atJp-0bZQ8H1aEjp9D0cHOkIlurifD5W/view?usp=drivesdk>

  ToggleChat-2.1.0-SNAPSHOT.jar
<https://drive.google.com/file/d/1clo_d0XN-n4XontjNt_1fggcozSd16RR/view?usp=drivesdk>

  Vanilla Enhancements-7.7.jar
<https://drive.google.com/file/d/1NRoNSPR2eYmsfCDy3DilATHrYVhbzGO8/view?usp=drivesdk>


  [1.8.9] Powns ToggleSneak Edit V1.1.jar
<https://drive.google.com/file/d/1Qw_V4_UN0gbLuUvC1NUmyL5iXD2uDFxd/view?usp=drivesdk>

**Comment by dxrthvxder** [comment]
Oh, well I was using 1.8.9 and loaded 20+ mods.

**Comment by HyCraftHD** [comment]
I don't really need the mods as the mods are likely not the cause of this issue.
But it still need the relevant log file of the 1.8.9 run.
Maybe ure using an other launcher etc. Look at the dates of the logfile. If the match when you run it then is should be the right one.

**Comment by dxrthvxder** [comment]
Yes, I know, I reloaded the launcher multiple times and I am on 1.8.9 with
the mod loaded. The log that I sent you was on 1.8.9 and everything should
be correct.

**Comment by HyCraftHD** [comment]
Then read the log and tell me where the log is 1.8.9 ?

**Comment by dxrthvxder** [comment]
I am sure it is 1.8.9, and even though it didn't say on the log here is a
screenshot showing I'm on 1.8.9

**Comment by HyCraftHD** [comment]
Please just use the github site. I cannot see any screenshot cuz of the email reply.
I believe you but then it's the wrong log file. Check the time of the log file with the time when you run your minecraft

**Comment by dxrthvxder** [comment]
Yea, ok.

**Comment by altrisi** [comment]
Can confirm that the language file is included but not loaded in the 1.8.9 jar, but last time I tested SoundCloud search it worked kind-of well (just results were unaccurate compared to searching in the official site/app).
I had to make a resourcepack with the language file itself to make it usable, then it would load the language strings from there but the mod won't display them by itself.
I won't send more feedback since I stopped using this mod today, just wanted to send this about the language since it can make the mod almost unusable for more users. Therefore I can't test this Soundcloud bug right now, but I will let some specific steps for the reporter to get the correct log.

If the bug still happens, run the game again and make sure you know the folder where the game is running (in case you changed the folder). Then try to reproduce the error, and once it have already happened, close the game and upload the latest.log file inside
`[Folder where it's running] > Logs > latest.log` using the Github website, since replying to the emails will usually remove the attachments. If you want and you find the specific error in the log you can also take a screenshot of that part, but the log is usually enough.
If the log you get still doesn't contain the MusicPlayer log (or like the previous it's from 1.12, check near the beginning of the file to make sure the version is correct), there's another thing you can try (that should reliably work), that is going to your launcher settings and enabling `Open output log when the game starts`. After that, run the game like normal (it will launch a log window, don't close it) and reproduce the bug. Then select all text in the log window, copy and paste it into a site like [Hastebin](https://hastebin.com/), save it (Ctrl+S or the button in the top right corner) and then paste the link here.

If you are interested I will upload my resourcepack as a workaround to fix the language error (the `gui.search` and others things instead of names) while it is being hopefully fixed.
[Fix for Music Player.zip](https://github.com/MC-U-Team/Music-Player/files/4753993/Fix.for.Music.Player.zip)

In case the issue no longer happens, well, either close the issue or consider changing the title to something like "Language file not loading for 1.8.9" so that other bug can also be tracked.

**Comment by HyCraftHD** [comment]
@dxrthvxder The search issue should be fixed in the latest version: 
![grafik](https://user-images.githubusercontent.com/7681220/84272181-5ff52200-ab2d-11ea-85a3-a2c55ab35a37.png)

The file issue should be fixed, at least in my test it worked. I didn't change anything so I cannot really know why it happens.
@altrisi Did you have any other mods installed? Also which forge version?

I'll close as the main issue is fixed.

**Comment by altrisi** [comment]
I'd say the problem of the language file shouldn't be about compatibility, but I was using Forge 1.8.9-11.15.1.1722 with the following mods:
```
.
│   256chat-1.8.9-1.1b.jar
│   Ambience+1.0-4.jar
│   Autotip-3.0.1 [1.8-1.12.2].jar
│   BetterFps-1.2.0.jar
│   Blur-MC1.8.9-1.0.4-2.jar
│   CustomMainMenu-MC1.8.9-2.0.jar
│   DynamicSurroundings-1.8.9-2.0.5.6.jar
│   foamfix-0.6.3-law-1.8.x.jar
│   FpsReducer-mc1.8.9-1.10.3.jar
│   MemoryFix-0.3.jar
│   MouseDelayFix-1.0.jar
│   noclosemychat-v1.0.jar
│   OptiFine_1.8.9_HD_U_L5.jar
│   Resource Exploit Fix-2.0 (1.8.9).jar
│   ResourceLoader-MC1.8.8-1.4.jar
│   SkyblockAddons-1.4.2-for-MC-1.8.9.jar
│   SneakAnimationFix-1.0.jar
│   [1.8.9] Sidebar Mod Revamp.jar
```
And of course the last MusicPlayer and UteamCore version (before this last one) and I was also previously using [TexFix](https://www.curseforge.com/minecraft/mc-mods/texfix) that I removed since I found it wasn't compatible with Optifine.

**Comment by altrisi** [comment]
Still happening for me in MusicPlayer 2.0.3.76, I'm opening a new issue.
